tests/components/cloud/test_account_link.py

```diff
@@ -152,9 +149,6 @@ def logger(self) -> logging.Logger:
 
     entry = hass.config_entries.async_entries(TEST_DOMAIN)[0]
 
-    assert (
-        await config_entry_oauth2_flow.async_get_config_entry_implementation(
-            hass, entry
-        )
-        is impl
-    )
+    assert (await
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)